scripts/gcloud.sh

```diff
@@ -165,117 +163,75 @@ gcloud_create_instance_template_if_not_exists() {
 # $5 - optional: password
 # $6 - optional: announce
 # $7 - optional: private key
-# $8 - optional: no args
-# $9 - optional: skip_update if exists already
 gcloud_create_or_update_instance_template() {
-  local args name mount_path image rpc api_token password host_path no_args private_key announce skip_update_if_exists
-  local extra_args=""
-
-  name="${1}"
-  image="${2}"
-  environment_id="${3:-}"
-
+  
+  local name="${1}"
+  local image="${2}"
+  local environment_id="${3:-}"
   # these parameters are only used by hoprd nodes
-  api_token="${4:-}"
-  password="${5:-}"
-
+  local api_token="${4:-}"
+  local password="${5:-}"
   # if set, let the node announce with a routable address on-chain
-  announce="${6:-}"
-
-  # this parameter is mostly used on by CT nodes, although hoprd nodes also
-  # support it
-  private_key="${7:-}"
-
-  # if set no additional arguments are used to start the container
-  no_args="${8:-}"
+  local announce="${6:-}"
+  # this parameter is mostly used on by CT nodes, although hoprd nodes also support it  
+  local private_key="${7:-}"
+  local mount_path="/app/hoprd-db"
+  local host_path="/var/hoprd"
+  local metadata_value=""
+  
+  log "checking for instance template ${name}"
+  if gcloud compute instance-templates describe "${name}" --quiet >/dev/null; then
+    log "instance template ${name} already present"
+    instance_group_name=$(echo $name | sed 's/\-[0-9].*//')    
```

Comment:
This works, but can use bash-ism: `${variable/search/replace}`

Reply:
Nice improvement